tailwind.config.ts

```diff
@@ -10,22 +10,25 @@ export default {
     },
     // https://www.figma.com/file/6M2LrpSCcB0thlFDaQAI2J/cx_jod_client?node-id=542%3A6241
```

Comment:
The link should be updated to the new version.

Reply:
Link updated